packages/polyfill/source/EventTarget.ts

```diff
@@ -1,14 +1,15 @@
-import {hooks} from './hooks.ts';
-import {LISTENERS} from './constants.ts';
+import {HOOKS, LISTENERS, OWNER_DOCUMENT} from './constants.ts';
 import {fireEvent, EventPhase} from './Event.ts';
 import {CAPTURE_MARKER, type Event} from './Event.ts';
 import type {ChildNode} from './ChildNode.ts';
+import type {Document} from './Document.ts';
 
 const ONCE_LISTENERS = Symbol('onceListeners');
 
 export class EventTarget {
   [LISTENERS]?: Map<string, Set<EventListenerOrEventListenerObject>>;
   [ONCE_LISTENERS]?: WeakMap<EventListenerOrEventListenerObject, EventListener>;
+  [OWNER_DOCUMENT]?: Document;
```

Comment:
Can we make this `private`? It might also be worth mentioning in a comment how this value is set (IIUC, it is only set on some subclasses, and we are relying on that behavior here).

Reply:
That's fair. IIRC these keywords are only for TypeScript and shouldn't have any effect at runtime, but I'm ok with adding it. In this case, it might be `protected` instead of `private` as these are inherited.